src/routes/clinical-proxy.ts

```diff
@@ -36,6 +35,26 @@ const handleError = (err: Error, req: Request, res: Response) => {
   return res.status(500).send('Internal Server Error');
 };
 
+router.use(
+  '/template/all',
+  createProxyMiddleware({
+    target: CLINICAL_SERVICE_ROOT,
+    pathRewrite: (pathName: string, req: Request) => {
+      const param = req.query.excludeSampleRegistration;
+      const exclude = param === undefined ? 'false' : param === 'true';
+      return urlJoin('/dictionary/template/all', `?excludeSampleRegistration=${exclude}`);
+    },
+    onError: handleError,
+    changeOrigin: true,
+    onProxyReq(proxyReq, req, res) {
+      const exclude = req.query.excludeSampleRegistration;
+      if (exclude !== 'true' && exclude !== 'false' && exclude !== undefined) {
```

Comment:
```
if(excludeRegistration) { ...
```

Reply:
hmm, this condition is to check invalid query param, in case other services like ui sends stuff other than `true` or `false` or `undefined`, gateway can quickly respond with an error without having to reach to clinical.